### PR TITLE
FLAG-59 : Enable patient flags to run in 2.2.0 platform

### PR DESCRIPTION
**Pull request by slubwama**
Enable patient flags to run in 2.2.0 platform
https://issues.openmrs.org/browse/FLAG-59

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java [hunk 1]
```diff
@@ -85,7 +87,9 @@ public void started() {
 		}
 		
 		// set the new names
-		thisModule.setExtensionNames(extensionPoints);
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: The method was not removed. The parameter was rather changed from `IdentityHashMap` to `Map`.
So, you need to deal with those two cases.

Comment (slubwama) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: @dkayiwa any examples available

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: Is this of help? https://github.com/openmrs/openmrs-module-xforms/commit/5d79f327d27777a40ec755a90433e4dc69067e4a

**Comment by slubwama** [comment]
@dkayiwa  done with reviews

File: api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java (hunk 1 above)
Comment (dkayiwa) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: Why don't you make it as simple as this? https://pastebin.com/0TMgGhKg

Comment (slubwama) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: @dkayiwa  thanks. I believe its done

File: api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java [hunk 2]
```diff
@@ -25,7 +28,9 @@
 import org.openmrs.module.BaseModuleActivator;
 import org.openmrs.module.Extension;
 import org.openmrs.module.Module;
+import org.openmrs.module.ModuleUtil;
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: Do you still need all these imports?

File: api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java (hunk 1 above)
Comment (dkayiwa) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: Does the solution make sense to you?

File: api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java (hunk 2 above)
Comment (slubwama) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: @dkayiwa done

Comment (dkayiwa) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: But the ticket says you are still doing more work on this.

File: api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java (hunk 1 above)
Comment (slubwama) [reply api/src/main/java/org/openmrs/module/patientflags/PatientFlagsModuleActivator.java]: yes it does. It was close.

**Comment by slubwama** [comment]
@dkayiwa done with reviews

**Comment by dkayiwa** [comment]
@slubwama the JIRA ticket shows that you are still doing more work on it.

**Comment by dkayiwa** [comment]
The ticket does not have a link to the pull request as required at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Pull request merged**